res/controllers/midi-components-0.0.js

```diff
@@ -318,21 +318,45 @@
         this.firstValueReceived = false;
     };
     Pot.prototype = new Component({
+        relative: false,
         inValueScale: function (value) { return value / this.max; },
-        input: function (channel, control, value, status, group) {
-            this.inSetParameter(this.inValueScale(value));
-            if (! this.firstValueReceived) {
-                this.firstValueReceived = true;
-                this.connect();
+        shift: function () {
+            if (this.relative) {
+                this.input = function (channel, control, value, status, group) {
+                    this.previousValueReceived = value;
+                }
+            }
+        },
+        unshift: function () {
+            this.input = function (channel, control, value, status, group) {
+                if (this.relative) {
+                    if (this.previousValueReceived !== undefined) {
+                        var delta = (value - this.previousValueReceived) / this.max;
+                        this.inSetParameter(this.inGetParameter() + delta);
```

Comment:
Can this exceed the maximum? Is this intended? I think in case of the rate slider it is allowed from the engine. 
The without a limit here, you cannot get back to the normal scale by moving the slider to the limit.